### ISSUE TITLE
[FR] New function: #blank-slide

### ISSUE THREAD

**Issue by KronosTheLate** [andreasKroepelin/polylux#140]
I am currently facing an issue where I want to insert a tall image. In powerpoint I would simply have it overlap with other elements, using the height. However, it appears as if all available slides (with the metropolis theme) clip the image, much above/below the bottom/top:
![image](https://github.com/andreasKroepelin/polylux/assets/61620837/1b30546b-a016-4a3e-973d-8c00a9776aab)

I think that this issue could be solved, and possibly many others, by creating a function `#blank-slide` for all themes. It would impose as little as possible (nothing?), and possibly have keyword-argument that allow e.g. incrementing the logical slide number.

**Comment by KronosTheLate** [comment]
I found what I was looking for: `#polylux-slide`. It however looks mainly intended for building templates. Should the documentation and name be adjusted to make it seem less hacky to use it for something like I am doing, and also to add discoverability?

**Comment by andreasKroepelin** [comment]
I agree that simply using `polylux-slide` is the way to go here. That's not hacky at all and the intended use. Themes (in the future, they will be known as templates because that's what they really are) just give you some more pre-existing scaffold for your slides. But if you don't want/need that and prefer a blank slide instead, then `polylux-slide` it is.

**Comment by KronosTheLate** [comment]
That makes a lot of sense. Perhaps it would suffice to just make a note about it in the docs? Me for example, was quite uncertain if this was hacky or more prone to breakage than other methods.

**Comment by andreasKroepelin** [comment]
I was hoping this first section of the docs where I deliberately don't use themes makes it clear that that's what you can do.

**Comment by KronosTheLate** [comment]
Yes, fair point. Perhaps I am just spoiled, expecting things to be very spelled out in terms of intended usage in possibilities. I had the idea that `#polylux-slide` was used for building themes, and that users should not expect to use it in a presentation with a theme.

Would you mind aliasing `blank-slide` to `polylux-slide`? I feel like it is more discoverable, and more clear in what it might be used for. I know I am being very nitpicky here, but I feel like it would be a marginal improvement without drawback.
